internal/scripts/scripts.go

```diff
@@ -33,6 +37,8 @@ type ContentValue struct {
 	RootDir    string
 	CheckRead  func(path string) error
 	CheckWrite func(path string) error
+	// Create should be called to write to file.
```

Comment:
Can you remove the comment? It is talking about how other piece of the code have to operate. And when https://github.com/letFunny/chisel/pull/8#discussion_r1519386200 is implemented the tests will guarantee that the scripts use the proper `Create` function.